### PR TITLE
changing to add hash to activate

### PR DESCRIPTION
**Pull request by stresslimit**
Update for new api interface add hash to activate

### CODE REVIEW

**Comment by stresslimit** [comment]
@jasonkuhrt i tested this out and it looks good, just need test to make more sense. I could keep it light and remove the failing test if you want.

**Comment by jasonkuhrt** [comment]
@houndci wtf machine, who invited you to the party?

@stresslimit Reviewing now.

**Comment by jasonkuhrt** [comment]
There are some other bugs in the tests that make no sense to me right now. Ugh. Still investigating.

File: lib/routes.js [hunk 1]
```diff
@@ -145,28 +319,10 @@ module.exports = [
...
-            {
-              "name": "label",
-              "type": "string",
-              "required": null,
-              "allowedValues": null,
-              "disallowedValues": null
-            }
-          ]
-        }
-      ]
```
Comment (jasonkuhrt) [inline lib/routes.js]: Bug here. Somehow our allowed values here were blown away.

**Comment by jasonkuhrt** [comment]
Currently failing cases:

- [ ] PUT https://api-http.littlebitscloud.cc/devices/a1 (understood)
- [ ] GET https://api-http.littlebitscloud.cc/subscriptions (not understood)
- [ ] DELETE https://api-http.littlebitscloud.cc/subscriptions (not understood)

**Comment by jasonkuhrt** [comment]
Ok, this is a certified mess. I am really going down a rabbit hole that I don't have the time to resolve right now. The code-debt on our routes generation has come nipping at our heels. I will try another appraoch where we manually edit the routes file for now.

**Comment by jasonkuhrt** [comment]
@stresslimit Ok this is ready to go. We have debt across the board. This is scary stuff. I made an issue https://github.com/littlebitselectronics/cloud-platform/issues/32 that could represent a path forward. I'm really unhappy with the state of things.

**Comment by stresslimit** [comment]
Yeah I hear you @jasonkuhrt. We gots lots to clean. No lack of things to refactor based on our learnings :)

**Pull request merged**